tests/cypress/integration/actions_objects2/case_115_ellipse_shape_track_label.js

```diff
@@ -78,8 +78,6 @@ context('Actions on ellipse.', () => {
                 (createEllipseTrackSwitchLabel.rightX + createEllipseTrackSwitchLabel.cx) / 2,
                 createEllipseTrackSwitchLabel.topY + 20,
                 '53.1',
-                false,
```

Comment:
Are you sure you want to remove ``pressShift`` argument?

Reply:
For this case, yes. The rotation check with shift occurs below.

Reply:
``pressShift`` does not have a default value, and initialized by ``undefined`` in this case. 
Good practice is pass this argument or set a default value for the parameter.

Reply:
Set ```false``` as the default value for the parameter.